beacon/goclient/goclient.go

```diff
@@ -132,11 +142,13 @@ func New(logger *zap.Logger, opt beaconprotocol.Options) (beaconprotocol.Beacon,
 
 	network := beaconprotocol.NewNetwork(core.NetworkFromString(opt.Network), opt.MinGenesisTime)
 	_client := &goClient{
-		ctx:            opt.Context,
-		network:        network,
-		client:         httpClient.(*http.Service),
-		indicesMapLock: sync.Mutex{},
-		graffiti:       opt.Graffiti,
+		log:               logger,
+		ctx:               opt.Context,
+		network:           network,
+		client:            httpClient.(*http.Service),
+		graffiti:          opt.Graffiti,
+		operatorID:        operatorID,
+		batchRegistration: true,
```

Comment:
it there a reason to not do batch registration? if so we should probably create a ticket/comment about adding configuration for this. otherwise let's remove this.

Reply:
The submission process is much simpler if not batched, so it might be useful for debugging if the registration-related code is suspected in having bugs. However, straightforward submission is quick to implement, so I agree and will get rid of this